### ISSUE TITLE
Teil can be masculine

### ISSUE THREAD

**Issue by dotlambda** [machiav3lli/DerDieDas#24]
Teil can be masculine when used as a synonym for Anteil.

**Comment by machiav3lli** [comment]
Yes, I got that. But in this situation (consider the translation)  it's specifically not.

**Comment by dotlambda** [comment]
It could be replaced by e.g. Bauteil, that would be unambiguous.

**Comment by guillaume-d** [comment]
See also the original issue https://github.com/digitapex/DerDieDas/issues/2.

Now having a translation makes it less misleading indeed, even if one could still think the word only can have that gender.
As already said there teaching wrong things should be avoided above all.

**Comment by mvhulten** [comment]
I have been checking gender (among many things) while improving the Dutch translation.  I used this approach (giving only the meaning that corresponds to the gender, having the list (`app/src/main/res/raw/list_nouns.txt`) next to my browser window).

I think in this way the words and genders get wired correctly in your brain.
